src/assets/JourneyWithGo-ABlogSeries/goControlFlow.md

```diff
@@ -0,0 +1,410 @@
+---
+title: Control Statements - Control Your Go
+authorName: Prarup Gurung (प्रारुप गुरुङ्ग)
+authorAvatar: https://www.jankaritech.com/images/2021/12/29/p1110938-crop-colour-800.jpg
+authorLink: https://github.com/grgprarup
+createdAt: Dec 13, 2022
```

Comment:
Dec 13, 2022? 🤔 
```suggestion
createdAt: Dec 13, 2022
```

Reply:
:smile: mmm that's the date when I started to write. I am also confused about which date we need to put (e.g. started date, first git commit date, PR approved date, PR merged date)? any suggestions?

Reply:
I don't think it's that big of an issue, but I would expect the date to be in 2023 somewhere around when PR was created and reviewed/approved(somewhere around the end of march or the beginning of April?)